Src/Plugins/Tests.Security/Validators/SEC101_017.NpmAuthorTokenValidatorTests.cs

```diff
@@ -20,37 +20,37 @@ public class NpmAuthorTokenValidatorTests
         [Fact]
         public void NpmAuthorTokenValidator_MockHttpTests()
         {
-            var testCases = new[]
+            var testCases = new HttpMockTestCase[]
             {
-                new
+                new HttpMockTestCase
                 {
                     Title = "Testing Unauthorized StatusCode",
-                    HttpStatusCode = HttpStatusCode.Unauthorized,
-                    HttpContent = (HttpContent)null,
+                    HttpStatusCodes = new[] {HttpStatusCode.Unauthorized },
```

Comment:
>HttpStatusCodes [](http://example.com/codeflow?start=20&length=15)

let's switch to httpresponsemessages, it will be clearer

Reply:
Done